node/buildutils.js

```diff
@@ -57,59 +61,86 @@ exports.getExternals = function () {
     }
 }
 
-var downloadFile = function (url) {
-    // validate parameters
-    if (!url) {
-        throw new Error('Parameter "url" must be set.');
-    }
+exports.getExternalsAsync = getExternalsAsync
 
-    // short-circuit if already downloaded
-    var scrubbedUrl = url.replace(/[/\:?]/g, '_');
-    var targetPath = path.join(downloadPath, 'file', scrubbedUrl);
-    var marker = targetPath + '.completed';
-    if (!test('-f', marker)) {
-        console.log('Downloading file: ' + url);
 
+
+/**
+ * @deprecated This method uses library which is not prefered to use on production
+ */
+
+exports.getExternals = function () {
```

Comment:
extra line. If it's still works it's ok then

Reply:
Extra line removed